vembrane/ann_types.py

```diff
@@ -308,29 +341,25 @@ def convert(self, key: str, value: str) -> Tuple[str, AnnotationType]:
         ",",
         description="List of cell types and classifications for regulatory feature",
     ),
-    # TODO parse flag:
-    #  "a flag indicating if the transcript is denoted as the canonical transcript
-    #  for this gene"
-    # "CANONICAL": AnnotationEntry(
-    #     "CANONICAL",
-    #     description="A flag indicating if the transcript is denoted as the canonical "
-    #                 "transcript for this gene",
-    # ),
+    "CANONICAL": AnnotationEntry(
+        "CANONICAL",
+        typefunc=lambda x: True if x == "YES" else False,
```

Comment:
(Same as above)
```suggestion
        typefunc=lambda x: if x == "YES",
```
Though, if you think `True if ... else False` is more readable, I'm happy to leave it as is!
If micro-optimization performance is of interest,
```suggestion
        typefunc="YES".__eq__,
```
;).